lib/browser/api/session.js

```diff
@@ -2,22 +2,7 @@
 
 const { EventEmitter } = require('events')
 const { app, deprecate } = require('electron')
-const { Session, Cookies } = process.atomBinding('session')
-const realFromPartition = process.atomBinding('session').fromPartition
-
-const wrappedSymbol = Symbol('wrapped-deprecate')
-const fromPartition = (partition) => {
-  const session = realFromPartition(partition)
-  if (!session[wrappedSymbol]) {
-    session[wrappedSymbol] = true
-    const { cookies } = session
-    cookies.flushStore = deprecate.promisify(cookies.flushStore)
-    cookies.get = deprecate.promisify(cookies.get)
-    cookies.remove = deprecate.promisify(cookies.remove)
-    cookies.set = deprecate.promisify(cookies.set)
-  }
-  return session
-}
+const { fromPartition, Session, Cookies, NetLog, Protocol } = process.atomBinding('session')
```

Comment:
I recognize the usefulness of this change but can it perhaps be done in a follow-up PR to prevent scope creep?